nasa-gibs/worldview
# WV-2627 mobile warning alignment

## Description

Fixes # wv-2627

On mobile, the warning ("!" in the triangle) is misaligned in the "Recent" tab.

![image](https://user-images.githubusercontent.com/19156576/227555489-ec4fdbc6-39ba-4a19-a6a6-dc3d66cc3a89.png)

## How To Reproduce
- Add Layers --> Recent --> Clear List --> close (this isn't required, but will make it easier to see the problem layers)
- Add Layers --> Search for "Reflectance" & scroll down until you find some with the warning mark
- Click to add a couple options
- Go to "Recent" to see the layers you selected with the warning misaligned

## How To Test
- Follow the steps above & confirm the warning mark is properly aligned

Note: This error seems to be caused by a difference in the installed react-swipe-to-delete package (0.5.4) & the CDN react-swipe-to-delete CSS (current version, 1.0.8). This updates react-swipe-to-delete to 1.0.5 & removes the CDN link in favor of a local copy of the current CSS. This may need to be revisited/confirmed when [wv-2578 ](https://github.com/nasa-gibs/worldview/pull/4148) is approved as that also bumps this package version.

@nasa-gibs/worldview
